README.md

```diff
@@ -1,3 +1,81 @@
 # loopback-cli
 
 LoopBack CLI tool for creating projects, models and more.
+This package supersedes the older `slc` tool.
+
+## Installation
+
+```
+$ npm install loopback-cli
+```
+
+## Usage
+
+### Getting started
+
+ 1. Run `lb` to create a new LoopBack application
+ 2. Run `node .` to start the scaffolded server
```

Comment:
I copied this from [generator-loopback's README](https://github.com/strongloop/generator-loopback/blob/3e1322a4b5fba7643f886f6272e5ae20f0d7e261/README.md). Does `npm start` provide any benefits compared to `node .`? I think the choice it's just a matter of personal preference.